scripts/docker/install-go.bash

```diff
@@ -17,7 +17,8 @@ set -e
 
 curl --silent https://dl.google.com/go/go1.13.4.linux-amd64.tar.gz | tar -C /usr/local -xzf -
 
-go get github.com/golang/protobuf/protoc-gen-go
+export GO111MODULE="on"
```

Comment:
please add comment why need to add this?

Reply:
I will do that.

Reply:
Thanks for spotting the missing installation on `protoc-gen-go`.

I am wondering why we need to `export GO111MODULE="on"`. On line 18, we downloaded Go 1.13, and Go module is enable by default for 1.13.

Reply:
It raised an error: `go: cannot use path@version syntax in GOPATH mode` without `GO111MODULE="on" `

Reply:
I see. Because `docker build` executed `install-go.bash` in the `/` directory, which is not a go package, so the `go get path@version` complaint.